src/tests/test_config.py

```diff
@@ -91,7 +91,44 @@ def test_start_ini_default_connection_if_any(tmp_empty, ip_no_magics):
     assert ConnectionManager.current.dialect == "sqlite"
 
 
-def test_start_ini_default_connection_using_pyproject_if_any(tmp_empty, ip_no_magics):
+def test_load_home_jupysql_toml_if_no_pyproject_toml(tmp_empty, ip_no_magics, capsys):
+    primary = Path("pyproject.toml")
+    if primary.exists():
+        primary.unlink()
+    alternate = Path("~/.jupysql/config").expanduser()
+    alternate.parent.mkdir(exist_ok=True)
+    if alternate.exists():
+        alternate.unlink()
```

Comment:
this will modify the user settings of whoever runs the tests, it'd be better to mock this.

you can mock the`expanduser` so the `~` resolves to a directory of your choosing in the current directory, instead of the actual user directory

Reply:
[fixed](https://github.com/ploomber/jupysql/pull/899/files#diff-c7a5329ac9b1ee1d1f5f3018031825611df32db9214cb42981f95243bdbd400bR94-R100)